FlowCrypt/src/main/java/com/flowcrypt/email/core/msg/MimeUtils.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors:
+ *   Ivan Pizhenko
+ */
+
+package com.flowcrypt.email.core.msg
+
+import com.flowcrypt.email.api.retrofit.response.model.node.MsgBlock
+import com.flowcrypt.email.api.retrofit.response.model.node.PlainAttMsgBlock
+import com.flowcrypt.email.extensions.kotlin.toInputStream
+import java.nio.charset.StandardCharsets
+import java.util.Locale
+import java.util.Properties
+import javax.mail.Session
+import javax.mail.internet.MimeMessage
+
+object MimeUtils {
+  fun resemblesMsg(msg: ByteArray?): Boolean {
+    if (msg == null) return false
+    val firstChars = msg.copyOfRange(0, msg.size.coerceAtMost(1000))
+      .toString(StandardCharsets.US_ASCII)
+      .toLowerCase(Locale.ROOT)
```

Comment:
I wonder if the conversion to String can fail with an exception if we give it arbitrary bytes that are non-ascii. If it can fail by throwing, we should catch the error and return false.

Reply:
Did the small test:
```kotlin
import java.nio.charset.StandardCharsets
import java.util.Locale

fun main() {
    val a = ByteArray(256)
    for (i in 0..255) a[i] = i.toByte()
    val s = a.toString(StandardCharsets.US_ASCII).toLowerCase(Locale.ROOT)
    println(s)
}
```
Works for all source byte values.